src/frontend/js/components/AshleyEditor/LatexButton/index.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { EditorState } from 'draft-js';
+import {
+  insertInlineTeX,
+  insertTeXBlock,
+} from 'draft-js-latex-plugin/lib/utils/insertTeX';
+import { TypeLatexStyle } from '../../../types/Enums';
+import { LatexButton } from '.';
+
+const props = {
+  editorState: new EditorState(),
+  type: TypeLatexStyle.INLINE,
+  getEditorState: jest.fn(),
+  setEditorState: jest.fn(),
+  theme: {},
+};
+jest.mock('draft-js-latex-plugin/lib/utils/insertTeX', () => ({
+  insertInlineTeX: jest.fn(),
+  insertTeXBlock: jest.fn(),
+}));
+
+describe('LatexButton', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+  beforeEach(() => {
+    props.editorState = EditorState.createEmpty();
+    props.getEditorState = jest.fn().mockReturnValue(props.editorState);
+  });
```

Comment:
Do you really need to reset those values ?

Reply:
at one point yes, but not after my resetMock and I didn't erased it, good catch